packages/jellyfish-transaction-signature/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "private": false,
+  "name": "@defichain/jellyfish-transaction-signature",
+  "version": "0.0.0",
+  "description": "A collection of TypeScript + JavaScript tools and libraries for DeFi Blockchain developers to build decentralized finance on Bitcoin",
+  "keywords": [
+    "DeFiChain",
+    "DeFi",
+    "Blockchain",
+    "API",
+    "Bitcoin"
+  ],
+  "repository": "DeFiCh/jellyfish",
+  "bugs": "https://github.com/DeFiCh/jellyfish/issues",
+  "license": "MIT",
+  "contributors": [
+    {
+      "name": "DeFiChain Foundation",
+      "email": "engineering@defichain.com",
+      "url": "https://defichain.com/"
+    },
+    {
+      "name": "DeFi Blockchain Contributors"
+    },
+    {
+      "name": "DeFi Jellyfish Contributors"
+    }
+  ],
+  "main": "dist",
+  "types": "dist",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc -b ./tsconfig.build.json"
+  },
+  "peerDependencies": {
+    "bignumber.js": "^9.0.1"
+  },
+  "dependencies": {
+    "@defichain/jellyfish-crypto": "0.0.0",
+    "@defichain/jellyfish-transaction": "0.0.0",
+    "smart-buffer": "^4.1.0"
+  },
+  "devDependencies": {
+    "@defichain/jellyfish-api-core": "0.0.0",
+    "@defichain/jellyfish-api-jsonrpc": "0.0.0",
+    "@defichain/testcontainers": "0.0.0"
+  }
```

Comment:
Same for this, I don't think we using it.

Reply:
They are used in unit test, wouldn't that affect?

Reply:
I don't think these 3 are used in the signature package. They are only used in builder tests?

Reply:
Yes, you're right. I was on false impression, as they were copied from jellyfish-transaction. Indeed, they were needed in the first place.